README.md

```diff
@@ -730,76 +730,130 @@ classDiagram
     AtRule <|.. KeyFrame: realization
     CSSList <|-- KeyFrame: inheritance
+    KeyFrame ..> OutputFormat: dependency
     Selector <|-- KeyframeSelector: inheritance
+    LineName ..> OutputFormat: dependency
+    LineName ..> ParserState: dependency
     ValueList <|-- LineName: inheritance
     SourceException <|-- OutputException: inheritance
     OutputFormat ..> OutputFormatter: dependency
+    OutputFormatter ..> Commentable: dependency
+    OutputFormatter ..> InvalidArgumentException: dependency
     OutputFormatter ..> OutputFormat: dependency
+    Parser ..> Document: dependency
     Parser ..> ParserState: dependency
+    ParserState ..> Anchor: dependency
+    ParserState ..> Comment: dependency
     ParserState ..> Settings: dependency
+    ParserState ..> SourceException: dependency
+    ParserState ..> UnexpectedEOFException: dependency
+    ParserState ..> UnexpectedTokenException: dependency
     Value <|-- PrimitiveValue: inheritance
+    Renderable ..> OutputFormat: dependency
     CSSElement <|.. Rule: realization
     Commentable <|.. Rule: realization
+    Rule ..> OutputFormat: dependency
+    Rule ..> ParserState: dependency
     Positionable <|.. Rule: realization
     Rule ..> RuleValueList: dependency
+    RuleContainer ..> Rule: dependency
     CSSElement <|.. RuleSet: realization
     CSSListItem <|.. RuleSet: realization
+    RuleSet ..> OutputFormat: dependency
+    RuleSet ..> ParserState: dependency
     Positionable <|.. RuleSet: realization
     RuleSet ..> Rule: dependency
     RuleContainer <|.. RuleSet: realization
+    RuleSet ..> UnexpectedValueException: dependency
     ValueList <|-- RuleValueList: inheritance
+    Selector ..> OutputFormat: dependency
     Renderable <|.. Selector: realization
+    Size ..> OutputFormat: dependency
+    Size ..> ParserState: dependency
     PrimitiveValue <|-- Size: inheritance
     Exception <|-- SourceException: inheritance
     Positionable <|.. SourceException: realization
     URL ..> CSSString: dependency
+    URL ..> OutputFormat: dependency
+    URL ..> ParserState: dependency
     PrimitiveValue <|-- URL: inheritance
     UnexpectedTokenException <|-- UnexpectedEOFException: inheritance
     SourceException <|-- UnexpectedTokenException: inheritance
     CSSElement <|.. Value: realization
+    Value ..> CSSFunction: dependency
+    Value ..> ParserState: dependency
     Positionable <|.. Value: realization
+    Value ..> RuleValueList: dependency
+    Value ..> UnexpectedTokenException: dependency
+    ValueList ..> OutputFormat: dependency
     Value <|-- ValueList: inheritance
-
-    CSSList ..> CSSList: dependency
-    CSSList ..> Comment: dependency
-    CSSList ..> RuleSet: dependency
-    CSSNamespace ..> Comment: dependency
-    Charset ..> Comment: dependency
-    Import ..> Comment: dependency
-    OutputFormat ..> OutputFormat: dependency
-    Rule ..> Comment: dependency
-    RuleSet ..> Comment: dependency
-    ValueList ..> Value: dependency
```

Comment:
`ValueList` extents `Value` and also has `Value`s as properties.  It looks like the latter connection is being lost.